package.json

```diff
@@ -40,6 +40,7 @@
     "@types/sinon-chai": "~3.2",
     "buble": "0.19.4",
     "chai": "~4.2",
+    "cpy-cli": "^2.0.0",
```

Comment:
Can we write this dependency as `~2.0`?

Reply:
Yes we can, but why should we?

Reply:
Offline discussion: moving to `~2.0` mostly due to being safer on package updates.